main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Love the use of the while loop. Maybe consider removing the parenthesis from the condition 
```suggestion
while loop < 9:
```